### PR TITLE
Add support many encoders

### PR DESCRIPTION
**Pull request by mdhsl**
Add support for various encoders:
- VP8
- VP9
- H265

1) Add names for selection into android APP
2) Create inheritance hierarchie to regroup common code
3) Fix bandwith by specify the framerate, otherwise it will try to take the maximium bandwith

### CODE REVIEW

File: sensorhub-driver-android/src/main/java/org/sensorhub/impl/sensor/android/video/AndroidCameraOutputH265.java [hunk 1]
```diff
@@ -39,16 +42,25 @@ public AndroidCameraOutputH265(AndroidSensorsDriver parentModule, int cameraId,
         super(parentModule,cameraId,previewTexture,"camera" + cameraId + "_H265");
     }
 
+    @Override
+    protected void initOutputStructure() {
+        // create SWE Common data structure and encoding
+        VideoCamHelper fac = new VideoCamHelper();
+        // use the one of H264
+        DataStream videoStream = fac.newVideoOutputH264(getName(), imgWidth, imgHeight);
```
Comment (alexrobin) [inline sensorhub-driver-android/src/main/java/org/sensorhub/impl/sensor/android/video/AndroidCameraOutputH265.java:50]: Need to update to set H265 as codec in output encoding description

File: sensorhub-driver-android/src/main/java/org/sensorhub/impl/sensor/android/video/AndroidCameraOutputVP8.java [hunk 2]
```diff
@@ -40,15 +43,25 @@ public AndroidCameraOutputVP8(AndroidSensorsDriver parentModule, int cameraId, S
     }
 
 
+    @Override
+    protected void initOutputStructure() {
+        // create SWE Common data structure and encoding
+        VideoCamHelper fac = new VideoCamHelper();
+        // use the one of H264
+        DataStream videoStream = fac.newVideoOutputH264(getName(), imgWidth, imgHeight);
```
Comment (alexrobin) [inline sensorhub-driver-android/src/main/java/org/sensorhub/impl/sensor/android/video/AndroidCameraOutputVP8.java:51]: Need to set VP8 in output encoding description

File: sensorhub-driver-android/src/main/java/org/sensorhub/impl/sensor/android/video/AndroidCameraOutputVP9.java [hunk 3]
```diff
@@ -62,15 +62,25 @@ public AndroidCameraOutputVP9(AndroidSensorsDriver parentModule, int cameraId, S
     }
 
 
+    @Override
+    protected void initOutputStructure() {
+        // create SWE Common data structure and encoding
+        VideoCamHelper fac = new VideoCamHelper();
+        // use the one of H264
+        DataStream videoStream = fac.newVideoOutputH264(getName(), imgWidth, imgHeight);
```
Comment (alexrobin) [inline sensorhub-driver-android/src/main/java/org/sensorhub/impl/sensor/android/video/AndroidCameraOutputVP9.java:70]: Need to set VP9 in output encoding description

File: sensorhub-driver-android/src/main/java/org/sensorhub/impl/sensor/android/video/AndroidCameraOutputH265.java (hunk 1 above)
Comment (mdhsl) [reply sensorhub-driver-android/src/main/java/org/sensorhub/impl/sensor/android/video/AndroidCameraOutputH265.java]: Done using generic descriptor

File: sensorhub-driver-android/src/main/java/org/sensorhub/impl/sensor/android/video/AndroidCameraOutputVP8.java (hunk 2 above)
Comment (mdhsl) [reply sensorhub-driver-android/src/main/java/org/sensorhub/impl/sensor/android/video/AndroidCameraOutputVP8.java]: Done using generic descriptor

File: sensorhub-driver-android/src/main/java/org/sensorhub/impl/sensor/android/video/AndroidCameraOutputVP9.java (hunk 3 above)
Comment (mdhsl) [reply sensorhub-driver-android/src/main/java/org/sensorhub/impl/sensor/android/video/AndroidCameraOutputVP9.java]: Done using generic descriptor

**Pull request merged**